src/SearchResults.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import moment from "moment";
+import CustomerProfile from "./CustomerProfile";
+import NewBookings from "./NewBookings";
+
+
+const getDaysDifference = (date1, date2) => {
+  let a = moment(date1);
+  let b = moment(date2);
+  return a.diff(b, 'days')
+
+}
+
+const SearchResults = (props) => {
+  const [selectedId, setselectedId] = useState(undefined);
+  const [clickIdProfile, setClickIdProfile] = useState(undefined);
```

Comment:
You dont actually need to put `undefined` into the `useState()` here, it is this by default